### PR TITLE
Move var folder in composer.json

### LINKED ISSUE #16: Var folder in composer.json
The `"symfony-var-dir": "web"` of the composer.json file should be set to `var`. Be aware of the the following files will move from `web` to `var`:
- bootstrap.php.cache
- SymfonyRequirements.php

### PR DESCRIPTION
**Pull request by wachterjohannes**
This PR enables the indexing of doctrine entities.

Additionally fixes https://github.com/sulu/sulu-minimal/issues/16

### CODE REVIEW

**Comment by danrot** [comment]
Should that be a hotfix or not? It's named hotfix, but the target branch is develop.

**Comment by wachterjohannes** [comment]
it could also be included in 1.3 release but it s not that important

File: app/config/config.yml [hunk 1]
```diff
@@ -69,6 +69,9 @@ swiftmailer:
 
 # MassiveSearch Configuration
 massive_search:
+    persistence:
+        doctrine_orm:
+            enabled: true
```
Comment (danrot) [inline app/config/config.yml]: Shouldn't that be the default value? Then we wouldn't have to add it to the configuration...

**Review by danrot — CHANGES_REQUESTED** [review]

File: app/config/config.yml (hunk 1 above)
Comment (wachterjohannes) [reply app/config/config.yml]: currently the default-value is false.

**Comment by danrot** [comment]
@wachterjohannes Could you please rebase? Then I would test and merge it.

**Comment by wachterjohannes** [comment]
@danrot is rebased.

File: app/config/config.yml (hunk 1 above)
Comment (danrot) [reply app/config/config.yml]: Actually it looks like you made the default value to true 😉 
https://github.com/sulu/sulu/blame/b81a56e6357924d1ba8fd060bacdf5571ef6a8d6/src/Sulu/Bundle/SearchBundle/DependencyInjection/SuluSearchExtension.php#L48

**Review by danrot — APPROVED** [review]

**Pull request merged**